desktop/cmp/filter/FilterChooser.js

```diff
@@ -45,19 +45,26 @@ export const [FilterChooser, filterChooser] = hoistCmp.withFactory({
     }
 });
 
-function getFilterOption(opt) {
+FilterChooser.propTypes = {
+    ...Select.propTypes
+};
```

Comment:
Hard to say, but it seems like it might well be worth it to doc the actual list of props we expect developers to be able to successfully set, assuming it's a very meaningfully limited subset of `Select` props.